projects/2020-02-fenix-webrender/summary.Rmd

```diff
@@ -0,0 +1,272 @@
+---
+title: "Webrender Fenix perf experiment"
+author: "tdsmith"
+date: '2020-02-19'
+output:
+  html_document:
+    toc: true
+    toc_float: true
+    self_contained: false
+---
+
+# Abstract
+
+The performance effects of enabling WebRender in the 3.2.0/3.2.1 release of Firefox Preview for Android ("Fenix")
+on Pixel 2/3 devices were measured with an A/B experiment.
+
+As expected, composite time increased. Page load time, checkerboarding time, and paint times all decreased.
```

Comment:
Please add 95% confidence ranges for the metric (e.g., page load time decreased by 4-17%). This helps for later quick referencing.

Reply:
Add that Content frame time (vsync) isn't reliable due to difference with "all-pings".

Reply:
I don't know what that means; can you clarify?

Reply:
oh, right. lol, I'd forgotten everything about this report